src/constructs/dns/dns-records.ts

```diff
@@ -26,16 +26,37 @@ export interface GuDnsRecordSetProps {
  */
 export class GuDnsRecordSet {
   constructor(scope: GuStack, id: string, props: GuDnsRecordSetProps) {
+    const { name, recordType, resourceRecords, ttl } = props;
+    const { stage } = scope;
+
+    // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition -- more `RecordType`s will be added soon!
```

Comment:
As `RecordType` has a single value (`CNAME`), the linter is suggesting this `if` is not necessary as it'll always be true. We'll be adding `TXT` and other records soon, so ignore the linter.